packages/plugins/AddressSearch/src/utils/searchMethods/getSearchMethod.ts

```diff
@@ -26,7 +26,7 @@ export const getMethodContainer = () => {
       return method
     }
     throw new Error(
-      `AddressSearch: The given type "${type}" does not define a valid searchMethod.`
+      `The given type "${type}" does not define a valid searchMethod.`
```

Comment:
```suggestion
      `@polar/plugin-address-search: The given type "${type}" does not define a valid searchMethod.`
```

Let's add/keep this here so configurators know where to check.

Reply:
> I have specifically not touched Error messages, as they sometimes are used in toasts. Changing them would blow the proportions of these changes. If you feel standardizing those are important as well, we can tackle that separately.